client/src/frontend/containers/Journals/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo } from "react";
+import { journalsAPI, requests } from "api";
+import { useTranslation } from "react-i18next";
+import CheckFrontendMode from "global/containers/CheckFrontendMode";
+import EntityCollectionPlaceholder from "global/components/entity/CollectionPlaceholder";
+import EntityCollection from "frontend/components/entity/Collection";
+import {
+  useFetch,
+  useListQueryParams,
+  useListFilters,
+  useFromStore
+} from "hooks";
+import CollectionNavigation from "frontend/components/CollectionNavigation";
+import HeadContent from "global/components/HeadContent";
+
+export default function JournalsListContainer() {
+  const filterReset = useMemo(() => ({ standaloneModeEnforced: "false" }), []);
+
+  const subjects = useFromStore({
+    requestKey: requests.feJournalSubjects,
+    action: "select"
+  });
+
+  const { pagination, filters, setFilters } = useListQueryParams({
+    initSize: 8,
```

Comment:
This seems to low. Can we make it match the per-page for projects?

Reply:
It looks like if you remove it, it uses the default set in `useListQueryParams`, which is `20`. That sounds like a good number to me 👍🏻